cli/admin/src/main/java/io/pravega/cli/admin/controller/ControllerCommand.java

```diff
@@ -50,7 +61,39 @@ protected Context createContext() {
         ClientConfig clientConfig = new ClientConfig();
         clientConfig.register(JacksonJsonProvider.class);
         clientConfig.property("sun.net.http.allowRestrictedHeaders", "true");
-        Client client = ClientBuilder.newClient(clientConfig);
+
+        Client client;
+
+        // If tls parameters are configured, set them in client
+        if (getCLIControllerConfig().isTlsEnabled()) {
+            KeyStore ks = null;
+            InputStream trustStore = null;
+            try {
+                trustStore = new FileInputStream(new File(getCLIControllerConfig().getTruststore()));
+                ks = KeyStore.getInstance("JKS");
+                ks.load(trustStore, null);
+                trustStore.close();
+
+            } catch (KeyStoreException e) {
+                output("The keystore file is invalid, the keystore type is not supported: " + e.toString());
```

Comment:
`output` supports formatted text. Please use that api.

Reply:
Used the api for formatted text.